src/js/script.js

```diff
@@ -0,0 +1,63 @@
+let arr = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, ''];
```

Comment:
Bad naming. 'arr' is unclear
 arr initialized here but below you reset it
 so just - let arr = [];

Reply:
About bad naming of arr you are right, thanks I have fixed it. Regarding initialization arr, I need have array with values for displaying content in each cell after loading web-page, that's why I can't assign it equal empty array.